src/app/pharmacists/components/prescription-list/prescription-list.component.html

```diff
@@ -119,20 +118,19 @@ <h2 fxLayout="row" fxLayoutAlign="start center" fxLayoutGap="10px">
                   </div>
                 </div>
               </div>
-              <div *ngIf="'estadoActual' in element" class="element-detail"
-                   [@detailExpand]="element._id == expandedElement?._id ? 'expanded' : 'collapsed'">
+              <div *ngIf="'estadoActual' in element" class="element-detail" [@detailExpand]="element._id == expandedElement?._id ? 'expanded' : 'collapsed'">
```

Comment:
La expresión  *ngIf=" ... in ..." no la esta tomando bien el navegador, lo marca como  error... modificar por *ngIf="element?.estadoActual"